testsuite/manualmode/src/test/resources/git-persistence/.ssh/README.txt

```diff
@@ -0,0 +1,34 @@
+#Command used to generate id_ec_test
+$ ssh-keygen -t ecdsa
```

Comment:
For the non-OpenSSH key pairs, it would be good to generate them dynamically if possible to avoid issues with the keys expiring at some point.